pkg/frontend/openshiftcluster_list.go

```diff
@@ -34,22 +36,23 @@ func (f *frontend) _getOpenShiftClusters(ctx context.Context, r *http.Request, c
 		prefix += "resourcegroups/" + vars["resourceGroupName"] + "/"
 	}
 
-	i, err := f.db.OpenShiftClusters.ListByPrefix(vars["subscriptionId"], prefix)
+	skipToken, err := f._getSkipToken(r.URL.String())
 	if err != nil {
 		return nil, err
 	}
 
-	var ocs []*api.OpenShiftCluster
+	i, err := f.db.OpenShiftClusters.ListByPrefix(vars["subscriptionId"], prefix, skipToken)
+	if err != nil {
+		return nil, err
+	}
 
-	for {
-		docs, err := i.Next(ctx)
-		if err != nil {
-			return nil, err
-		}
-		if docs == nil {
-			break
-		}
+	docs, err := i.Next(ctx, 10)
```

Comment:
Surely we want this configurable (rather than hard-coded to 10)?

Reply:
Not right now for the frontend.  There may be flexibility and reason to do things slightly differently for the list all clusters Geneva Action, let's look at that when you get there.